### PR TITLE
Feature/list command

### PR DESCRIPTION
**Pull request by Defkon1**
**Major changes:**

- Added `-l|--list` command

**Minor changes:**
- updated README

Requires classes from #3

### CODE REVIEW

File: src/adr/Program.cs [hunk 1]
```diff
@@ -44,14 +44,23 @@ private static void Main(string[] args)
...
+
                 command.OnExecute(() =>
                 {
+                    var docFolder = AdrSettings.Current.DocFolder;
 
-                    app.Out.WriteLine("Command not implemented");
+                    AdrManager manager = new AdrManager(System.IO.Path.GetFullPath(docFolder));
 
-                    return (int)ExitCode.NotImplemented;
+                    var records = manager.GetList();
```
Comment (CommCody) [inline src/adr/Program.cs]:
```suggestion
                    var records = log.GetRecords();
```

File: src/adr/Program.cs [hunk 2]
```diff
@@ -44,14 +44,23 @@ private static void Main(string[] args)
...
-                command.Description = "(Command not implemented)";
+                command.Description = "Show a list of ADRs";
                 command.HelpOption(HelpOption);
+
                 command.OnExecute(() =>
                 {
+                    var docFolder = AdrSettings.Current.DocFolder;
 
-                    app.Out.WriteLine("Command not implemented");
+                    AdrManager manager = new AdrManager(System.IO.Path.GetFullPath(docFolder));
```
Comment (CommCody) [inline src/adr/Program.cs]:
```suggestion
                    var log = new ArchitectureDecisionLog(System.IO.Path.GetFullPath(docFolder));
```

**Review by CommCody — CHANGES_REQUESTED** [review]
These changes will only work after #3 is adapted

**Review by CommCody — APPROVED** [review]
I'm sorry, but some time was available and I wanted to try to help you finish some of this work.
To integrate this work with the one we finished in #3 I rebased this branch on main, so you might have to reset your local branch

Let me know if you think this is done, or you want to add some more ?

**Comment by Defkon1** [comment]
I solved conflicts on my branch and renamed the ex-AdrManager file to reflect the new class name.

I think this can be merged as well

**Pull request merged**